{{cookiecutter.repo_name}}/tox.ini

```diff
@@ -0,0 +1,50 @@
+[tox]
+envlist = {py27,py35}-django{18,19}
+
+[pep8]
+max-line-length = 120
+
+[pytest]
+DJANGO_SETTINGS_MODULE = test_settings
+addopts = --cov {{ cookiecutter.app_name }} --cov-report term-missing
+norecursedirs = .* docs requirements
+
+[testenv]
+deps =
+    django18: Django==1.8.14
+    django19: Django==1.9.8
```

Comment:
Definitely should add 1.10, but I'm a little conflicted on ranges:

*  Django wouldn't get upgraded to a higher version in the range for local test environments until another dependency is changed, triggering a rebuild of the virtualenv
* A minor incompatibility in a new Django point release could cause test runs to suddenly start failing even though you haven't made any code changes or explicitly updated any dependencies.

I guess that's acceptable, since Django really tries to avoid breaking compatibility in minor releases, and the odds of going in to upgrade a pinned Django requirement without updating any other dependency versions are pretty low.